### PR TITLE
Add OpReplaceAnyElements to ops_common

### PR DESCRIPTION
**Pull request by sivanravidos**
- Added OpReplaceAnyElements to opps_common. This op allow replacing any set of tokens with a given token.
This is useful for example when one wants to ingore a set of tokens, e.g. when applying a metric that should ignore all special tokens.
- Added unit test for the new op and the former OpReplaceElements

### CODE REVIEW

File: fuse/data/ops/ops_common.py [hunk 1]
```diff
@@ -702,3 +702,61 @@ def __call__(
             raise Exception(f"Unsupported object type {type(input_obj)}")
 
         return sample_dict
+
+
+class OpReplaceAnyElements(OpBase):
+    """
+    Replace any element from a given list of elements with a requested value
```
Comment (mosheraboh) [inline fuse/data/ops/ops_common.py:709]: list./ numpy array or tensor

File: fuse/data/ops/ops_common.py [hunk 2]
```diff
@@ -702,3 +702,61 @@ def __call__(
...
+    This is useful, for example, when converting a set of token IDs to -100 to make a metric ignore certain elements (e.g. all special tokens).
+    """
+
+    def __call__(
+        self,
+        sample_dict: NDict,
+        key_in: str,
+        find_any_val: List[Any],
+        replace_with_val: Any,
+        key_out: str = None,
```
Comment (mosheraboh) [inline fuse/data/ops/ops_common.py:721]: Optional[str]

**Review by mosheraboh — APPROVED** [review]
Looks good.
Minors inline.

File: fuse/data/ops/ops_common.py (hunk 1 above)
Comment (sivanravidos) [reply fuse/data/ops/ops_common.py]: fixed

File: fuse/data/ops/ops_common.py (hunk 2 above)
Comment (sivanravidos) [reply fuse/data/ops/ops_common.py]: fixed

**Pull request merged**